api/chaos/experimentation/v1/experimentation.proto

```diff
@@ -3,55 +3,16 @@ syntax = "proto3";
 package clutch.chaos.experimentation.v1;
 
 import "google/api/annotations.proto";
+import "google/protobuf/any.proto";
 import "validate/validate.proto";
 
 import "api/v1/annotations.proto";
 
 option go_package = "experimentationv1";
 
-message TestSpecification {
-  oneof config {
-    AbortFault abort = 2;
-    LatencyFault latency = 3;
-  }
-}
-
 message Experiment {
   fixed64 id = 1;
-  TestSpecification test_specification = 2;
-}
-
-// Targets requests from downstream_cluster -> upstream_cluster
-message ClusterPairTarget {
-  // The name of the downstream cluster
-  string downstream_cluster = 1 [ (validate.rules).string = {min_bytes : 1} ];
-
-  // The name of the upstream cluster
-  string upstream_cluster = 2 [ (validate.rules).string = {min_bytes : 1} ];
-}
-
-message AbortFault {
-  oneof target {
-    ClusterPairTarget cluster_pair = 1;
-  }
-
-  // The percentage of requests that will be faulted.
-  float percent = 2 [ (validate.rules).float = {gt : 0.0, lte : 100.0} ];
-
-  // The abort HTTP status that will be returned.
-  int32 http_status = 3 [ (validate.rules).int32 = {gt : 99, lt : 600} ];
-}
-
-message LatencyFault {
-  oneof target {
-    ClusterPairTarget cluster_pair = 1;
-  }
-
-  // The percentage of requests that will be slowed down.
-  float percent = 2 [ (validate.rules).float = {gt : 0.0, lte : 100.0} ];
-
-  // The latency duration in milliseconds.
-  int32 duration_ms = 3 [ (validate.rules).int32.gt = 0 ];
+  google.protobuf.Any test_config = 2;
```

Comment:
I'd remove `test` from the name of the field and call it either `spec` or `config`.

Reply:
yea, the name should be clear enough even without a `test` prefix - will update